CMakeLists.txt

```diff
@@ -113,87 +130,92 @@ set(CMAKE_C_COMPILER $ENV{CMAKE_C_COMPILER})
 set(BUILDDIR "$ENV{BUILDDIR}")
 
 if(ARCH_TYPE_OPTION)
-  if(NOT
-     (${ARCH_TYPE_OPTION} STREQUAL "x86")
-     AND NOT (${ARCH_TYPE_OPTION} STREQUAL "armv7"))
-    message(
-      AUTHOR_WARNING
-        "HW architecture is not defined, using x86. Allowed values are x86/armv7 (case sensitive)"
-      )
-    set(ARCH_TYPE_OPTION "x86")
-  endif()
+    if(
+        NOT
```

Comment:
@MaximRybak `if(NOT (${ARCH_TYPE_OPTION} STREQUAL "x86")`

Reply:
@BSolonenko You can avoid moving arguments to new line. Property **"max_subargs_per_line"** responsible for this. If set that value as _5_, arguments won't move to new line, if whole length of the arguments less then **"line_width"** value and amount of them don't more then _5_.

Reply:
@MaximRybak please make this default

Reply:
@dkandyba please make this default